test/integration/framework/perf_utils.go

```diff
@@ -37,6 +37,7 @@ type IntegrationTestNodePreparer struct {
 	countToStrategy []testutils.CountToStrategy
 	nodeNamePrefix  string
 	nodeSpec        *v1.Node
+	preparedNodes   []*v1.Node
```

Comment:
We don't need to store `v1.Node`.
I think storing names of Node should be enough.

```suggestion
	// preparedNodes have the name of nodes created by this IntegrationTestNodePreparer.
	preparedNodes   []string
```

(Please change the wording as you like, I'm not good at English so. 😅 )

Reply:
updated.